pkg/apis/config/v1alpha1/types.go

```diff
@@ -0,0 +1,20 @@
+package v1alpha1
+
+import (
+	healthcheckconfigv1alpha1 "github.com/gardener/gardener/extensions/pkg/apis/config/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ControllerConfiguration configuration resource
+type ControllerConfiguration struct {
+	metav1.TypeMeta `json:",inline"`
+
+	DefaultDevicePattern *string `json:"defaultDevicePattern,omitempty"`
+	DefaultHostWritePath *string `json:"defaultHostWritePath,omitempty"`
```

Comment:
Please provide documentation and markers that it's optional. The documentation should then also be added to the internal type representation.